Management/src/ActuatorApi/ActuatorApi.http

```diff
@@ -0,0 +1,114 @@
+@Actuator_Context=actuator
+#@Actuator_Context=cloudfoundryapplication
+
+### Get the default day-count forecast starting from today
+GET {{HostAddress}}/weatherforecast/
+Accept: application/json
+
+###  Get the default day-count forecast starting from a certain date
+GET {{HostAddress}}/weatherforecast?fromDate=12/8/2024
+Accept: application/json
+
+###  Get a specified-number-of-days forecast starting from today
+GET {{HostAddress}}/weatherforecast?days=2
+Accept: application/json
+
+### Get all forecast data
+GET {{HostAddress}}/AllForecastData
+Accept: application/json
+
+### ACTUATORS
+### Check CORS
+OPTIONS {{ManagementAddress}}/{{Actuator_Context}}
+access-control-request-method: GET
+access-control-request-headers: authorization
+origin: https://your-apps-manager.origin
+
+### Hypermedia (list all actuators)
+GET {{ManagementAddress}}/{{Actuator_Context}}
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Beans actuator, list registered services ("Beans" comes from the Spring world)
+GET {{ManagementAddress}}/{{Actuator_Context}}/beans
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### DB Migrations actuator, list pending and applied database migrations
+GET {{ManagementAddress}}/{{Actuator_Context}}/dbmigrations
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Env actuator, list environment/configuration sources and data
+GET {{ManagementAddress}}/{{Actuator_Context}}/env
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Info actuator
+GET {{ManagementAddress}}/{{Actuator_Context}}/info
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Health actuator
+GET {{ManagementAddress}}/{{Actuator_Context}}/health
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Health actuator, check a specified group of contributors
+GET {{ManagementAddress}}/{{Actuator_Context}}/health/readiness
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Loggers actuator, list all loggers
+GET {{ManagementAddress}}/{{Actuator_Context}}/loggers
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Loggers actuator, update minimum level for a logger  
```

Comment:
```suggestion
### Loggers actuator, update minimum level for a logger
```
(remove trailing whitespace, which was introduced in the last commit)

Reply:
Well that's mildly annoying, I did a straight copy/paste, must have auto-formatted it in there somehow